SomyaSinghal/Day1/01.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+
+int main() {
+    char name[50];
+    printf("Hello! What's your name? ");
+    // Your code to get the name input here.
+    scanf("%[^\n]s\n", &name);
```

Comment:
s\n after is not required and the \n you provide here got a input to next scanf that's why it will not ask any input for next question.